source/_integrations/nextcloud.markdown

```diff
@@ -34,6 +34,11 @@ nextcloud:
 
 ```
 
+## Backup Home Assistant to Nextcloud
+
+Nextcloud users can also schedule automated Home Assistant backups to Nextcloud using [Sebclem's](https://github.com/Sebclem) [Nextcloud Backup Add-On](https://github.com/Sebclem/hassio-nextcloud-backup).
```

Comment:
While cool, this is not supported by Home Assistant, thus should not be part of the official documentation.

Please use our community forum to share these types of things.

Thanks 👍